packages/frontend-core/src/components/grid/layout/Grid.svelte

```diff
@@ -109,8 +109,13 @@
     notifyError,
     buttons,
   })
-  $: minHeight =
-    Padding + SmallRowHeight + $rowHeight + (showControls ? ControlsHeight : 0)
+
+  // Derive min height and make available in context
+  const minHeight = derived(rowHeight, $height => {
```

Comment:
Small change to make this a derived store rather than a primitive, so it's observable.